db/migrate/20210511212929_change_to_single_iteration_publiushing.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeToSingleIterationPubliushing < ActiveRecord::Migration[6.1]
```

Comment:
You'll probably squash all the migrations anyway, but 🤷 

```suggestion
class ChangeToSingleIterationPublishing < ActiveRecord::Migration[6.1]
```